src/layouts/caregiverDashboard/caregiverDashboard.js

```diff
@@ -0,0 +1,18 @@
+import './caregiverDashboard.css'
+import Header from '../../components/header/header.js'
+import Title from '../../components/title/titleWelcome.js'
+import DropdownSearchbar from '../../components/dropdownSearchbar/dropdownSearchbar.js'
+import Table from '../../components/table/table'
+
+function CaregiverDashboard() {
+  return (
+    <div className="body">
```

Comment:
I'm not a huge fan of `body` as a CSS class name as it's also a HTML element. Perhaps `Container` would be better suited?